prometheus/promhttp/instrument_client.go

```diff
@@ -78,7 +78,6 @@ func InstrumentRoundTripperCounter(counter *prometheus.CounterVec, next http.Rou
 				1,
 				rtOpts.getExemplarFn(r.Context()),
 			)
-			counter.With(labels(code, method, r.Method, resp.StatusCode, rtOpts.extraMethods...)).Inc()
```

Comment:
Thinking loudly: I think the function name `exemplarAdd()` doesn't make it obvious the metric is incremented. Maybe renaming it to `addWithExemplar()` may clarify it?

Reply:
Agreed! My goal here was to fix the bug with as few changes as possible so I didn't consider changing the function name... IMO that'd best be deferred to a separate PR.

Reply:
https://github.com/prometheus/client_golang/issues/1120

Reply:
👍🏽 Thanks for noticing this